mars/core/operand/core.py

```diff
@@ -39,6 +40,11 @@
 _op_type_to_size_estimator: Dict[Type[OperandType], Callable] = dict()
 
 
+op_executed_number = Metrics.counter(
+    "mars.operand.executed_number", "The number of executed operands.", ("op",)
```

Comment:
The number of *what* kind of objects executed? Please make the name of the counter and the variable more explicit.

Reply:
How about `mars.operator_executed_number`? And the description is `The number of executed operators.`

Reply:
executed_operator_number

Reply:
Sorry but I've ignored the prefix.. Already merged.